README.md

```diff
@@ -1,9 +1,18 @@
 # platoon
 Experimental multi-GPU mini-framework for Theano
 
+It support **data-parallelism** inside one compute node, not
+model-parallelism. For model-parallelism check [Theano multiple GPUs
+tutorial](http://deeplearning.net/software/theano/tutorial/using_multi_gpu.html).
+
 This framework is still a prototype. It's interface is not polished and it is
 likely to undergo changes in the future.
 
+The framework allow multiple data-parallel algorithmes, but only
+[EASGD](http://arxiv.org/abs/1412.6651) is currently implemented.
+
+There is working examples in the examples directory.
```

Comment:
I did the typo fix and clarified the sentence. Can you check the
clarification?

On Thu, Jan 7, 2016 at 3:04 PM, Bart van Merriënboer <
notifications@github.com> wrote:

> In README.md
> <https://github.com/mila-udem/platoon/pull/14#discussion_r49119023>:
>
> >  This framework is still a prototype. It's interface is not polished and it is
> >  likely to undergo changes in the future.
> >
> > +The framework allow multiple data-parallel algorithmes, but only
> > +[EASGD](http://arxiv.org/abs/1412.6651) is currently implemented.
> > +
> > +There is working examples in the examples directory.
>
> is are
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mila-udem/platoon/pull/14/files#r49119023>.
>